internal/pkg/api/auth.go

```diff
@@ -83,14 +84,14 @@ func authAgent(r *http.Request, id *string, bulker bulk.Bulk, c cache.Cache) (*m
 	start := time.Now()
 
 	// authenticate
-	key, err := authApiKey(r, bulker, c)
+	key, err := authAPIKey(r, bulker, c)
 	if err != nil {
 		return nil, err
 	}
 
 	w := log.With().
 		Str(LogAccessAPIKeyID, key.Id).
-		Str(EcsHttpRequestId, r.Header.Get(logger.HeaderRequestID))
+		Str(EcsHTTPRequestID, r.Header.Get(logger.HeaderRequestID))
```

Comment:
Should this be `ECSHTTPRequestID` since Ecs is the Elastic Common Schema?
I presume `Http` and `Api` are detected because it's coming from a well known dictionary.

Reply:
I'll add it to the linter